packages/core/src/behaviors.ts

```diff
@@ -295,28 +308,19 @@ export function createPromiseBehavior<T, TEvent extends EventObject>(
           resolvedPromise.then(
             (response) => {
               if (!canceled) {
-                parent?.send(
```

Comment:
we need a test to verify that it's possible to send an event to a parent from within that start signal handler

Reply:
How would we test this?

Reply:
I probably meant a more general situation - not specific to a promise behavior. But also, this has been refactored for now so let's not bother with it.